setup.php

```diff
@@ -92,6 +92,34 @@ public function enqueue_scripts($page_hook) {
     );
     wp_enqueue_script('yoast-cmb2-plugin-js');
   }
+
+
+  /**
+   * Test whether Yoast SEO is hidden either by choice of the admin or because
+   * the post type is not a public post type
+   *
+   *
+   * @param  string $post_type (optional) The post type to test, defaults to the current post post_type.
+   *
+   * @return  bool        Whether or not the yoast seo metabox should be hidden
+   */
+  private function check_if_yoast_seo_is_hidden($post_type = null) {
+    $global_post = $GLOBALS['post'];
+
+    if (!isset($post_type) && (isset($global_post) && (is_object($global_post)
+      && isset($global_post->post_type)))) {
+      $post_type = $global_post->post_type;
+    }
+
+    if ( isset( $post_type ) ) {
+      // Don't make static as post_types may still be added during the run.
+      $cpts    = get_post_types(array('public' => true),'names');
```

Comment:
could we use the newer array style syntax here `['public' => true]` instead of the `array` keyword? Could we add a space after the comma too?